packages/storybook/.storybook/main.js

```diff
@@ -15,10 +15,13 @@ module.exports = {
       name: '@storybook/addon-essentials',
       options: {
         actions: false,
-        controls: false
+        controls: false,
+        measure: false,
+        outline: false,
+        backgrounds: false,
+        viewport: false
       }
-    },
-    '@storybook/addon-controls'
```

Comment:
this is removing all controls on Canvas, so possibility for users to play with our components/composites

Reply:
oh i must have accidentally deleted it. Thanks ill put that back in!